### PR TITLE
Added unit tests for avi API utils

### PR DESCRIPTION
**Pull request by chaitanyaavi**
##### SUMMARY

Added unit tests for avi API utils

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- test/units/module_utils/network/avi/test_avi_api_utils.py 

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by grastogi23 — APPROVED** [review]
+shipit

File: test/units/module_utils/network/avi/test_avi_api_utils.py [hunk 1]
```diff
@@ -0,0 +1,701 @@
...
+        existing_obj = {
+            'lb_algorithm': 'LB_ALGORITHM_LEAST_CONNECTIONS',
+            'use_service_port': False,
+            'server_auto_scale': False,
+            'host_check_enabled': False,
+            'enabled': True,
+            'capacity_estimation': False,
+            'fewest_tasks_feedback_delay': 10,
+            '_last_modified': '1471377748747040',
+            'cloud_ref': 'https://10.10.25.42/api/cloud/cloud-afe8bf2c-9821-4272-9bc6-67634c84bec9',
```
Comment (Qalthos) [inline test/units/module_utils/network/avi/test_avi_api_utils.py]: Please don't use routable IP addresses in tests, even if they are never used. Documentation ranges (https://tools.ietf.org/html/rfc5737) help ensure that this is clear to readers.

File: test/units/module_utils/network/avi/test_avi_api_utils.py [hunk 2]
```diff
@@ -0,0 +1,701 @@
...
+            'max_concurrent_connections_per_server': 0,
+            'url': 'https://10.10.25.42/api/pool/pool-20084ee1-872e-4103-98e1-899103e2242a',
+            'tenant_ref': 'https://10.10.25.42/api/tenant/admin',
+            'uuid': 'pool-20084ee1-872e-4103-98e1-899103e2242a',
+            'connection_ramp_duration': 10}
+
+        diff = avi_obj_cmp(obj, existing_obj)
+        assert diff
+        x = not diff
+        assert not x
```
Comment (Qalthos) [inline test/units/module_utils/network/avi/test_avi_api_utils.py]: I'm really not sure what this is supposed to do. You've proved that `diff` is truthy above, and now you're asserting that whatever object `diff` is is symmetric over `not`, but I'm not sure why. You don't seem to be making any claims about the contents of `diff`, so I don't quite see what the value of this test is.

**Review by Qalthos — CHANGES_REQUESTED** [review]

File: test/units/module_utils/network/avi/test_avi_api_utils.py (hunk 1 above)
Comment (chaitanyaavi) [reply test/units/module_utils/network/avi/test_avi_api_utils.py]: Updated all the ips

File: test/units/module_utils/network/avi/test_avi_api_utils.py (hunk 2 above)
Comment (chaitanyaavi) [reply test/units/module_utils/network/avi/test_avi_api_utils.py]: Removed unnecessary code

**Review by grastogi23 — APPROVED** [review]
+shipit

**Review by Qalthos — APPROVED** [review]

**Pull request merged**